### ISSUE TITLE
ReSharper and MoqComplete versioning

### ISSUE THREAD

**Issue by square-unit** [Abc-Arbitrage/Abc.MoqComplete#22]
Hi guys,

I'm a support engineer of the ReSharper team. The latest MoqComplete version is marked as compatible with all the versions newer than Wave 203 `Wave:203.0.0 ` However, it requests for Wave 203 components when being installed into Wave 211 (2021.1 EAP)

So it is available in R# Extension Manager despite it causes the related errors during the installation and when it's running in ReSharper.

>“Error resolving type JetBrains.ReSharper.Feature.Services.CSharp.Analyses.Bulbs.ICSharpContextActionDataProvider from assembly “JetBrains.ReSharper.Feature.Services.CSharp, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”, request originating from assembly “JetBrains.ReSharper.Feature.Services.CSharp, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”. Could not find the type by its FQN. Is assembly qualification specified? True! Is it of our own assembly? True! Is looking into referenced assemblies allowed? False! Was the qualified referenced assembly encountered during lookup? False! Tried loading an implicitly-referenced assembly? False! Were following a forwarder exported type's assembly ref? No!”

Please fix the versioning issue and upload the updated version with properly specified supported R# version: `Wave:[203.0.0] `

**Comment by lconstan** [comment]
Thanks for reaching out!
This commit (908e5c5699e0f25d4723c277f635810e9c6a300f) should fix the issue. A new version has been uploaded (1.4.5)

Please reopen if needed

**Comment by pinkfloydx33** [comment]
FYI to anyone stumbling across this: even though newer versions have been pushed to the Gallery, it seems that R# will still display the older versions of this extension that "satisfy" the current Wave version. Specifically, version 1.4.4 still shows in R# Extensions as available to install in 2021.3 EAP, despite A. having been superseded by several versions of the extension and B. not actually supporting Wave 213. 

In short, don't install Moq Complete (<1.4.5) into the EAP as its (unfortunately) not yet supported.
